app/shared/ui-foundation/src/desktopMain/kotlin/platform/PlatformWindow.desktop.kt

```diff
@@ -9,22 +9,39 @@
 
 package me.him188.ani.app.platform
 
+import androidx.compose.runtime.derivedStateOf
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.geometry.Rect
+import androidx.compose.ui.window.WindowPlacement
 import androidx.compose.ui.window.WindowScope
+import androidx.compose.ui.window.WindowState
 import me.him188.ani.app.platform.window.TitleBarWindowProc
+import me.him188.ani.utils.platform.Platform
+import me.him188.ani.utils.platform.isWindows
 
 actual open class PlatformWindow(
     val windowHandle: Long,
-    val windowScope: WindowScope? = null
+    val windowScope: WindowScope? = null,
+    val windowState: WindowState,
+    val platform: Platform
 ) {
     internal var savedWindowsWindowState: SavedWindowsWindowState? = null
 
     internal var titleBarWindowProc by mutableStateOf<TitleBarWindowProc?>(null)
 
-    var isUndecoratedFullscreen by mutableStateOf(false)
+    internal var isWindowsUndecoratedFullscreen by mutableStateOf(false)
```

Comment:
禁止外面 read, 只能 set 吧

```suggestion
    private var isWindowsUndecoratedFullscreen by mutableStateOf(false)
    
    fun setWindowsUndecoratedFullscreen(value: Boolean)
```

否则外面会能同时看到 `isUndecoratedFullscreen` 和 `isWindowsUndecoratedFullscreen`.